<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findCirclesApplicationClass</class>
 <widget class="QMainWindow" name="findCirclesApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1367</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>findCirclesApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bthSetPath">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>设置路径</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="labelTotalNum">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string>图片总数</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editTotalNum">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>30</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>25</width>
                <height>35</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="labelImgNum">
              <property name="text">
               <string>图片序号</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editImgNum">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnReadImg">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>读取图片</string>
          </property>
          <property name="icon">
           <iconset resource="findcirclesapplication.qrc">
            <normaloff>:/读取图片/tupian/读取图片/pictures_55.178571428571px_1204563_easyicon.net.png</normaloff>:/读取图片/tupian/读取图片/pictures_55.178571428571px_1204563_easyicon.net.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="editThresholdValue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetThresholdValue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>设置阈值</string>
            </property>
            <property name="icon">
             <iconset resource="findcirclesapplication.qrc">
              <normaloff>:/设置阈值/tupian/设置阈值/electrical_threshold_48px_1169345_easyicon.net.png</normaloff>:/设置阈值/tupian/设置阈值/electrical_threshold_48px_1169345_easyicon.net.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLineEdit" name="editPixelThreshold">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPixelThreshold">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>设置亮度值</string>
            </property>
            <property name="icon">
             <iconset resource="findcirclesapplication.qrc">
              <normaloff>:/设置亮度值/tupian/设置亮度阈值/Brightness_48px_1191655_easyicon.net.png</normaloff>:/设置亮度值/tupian/设置亮度阈值/Brightness_48px_1191655_easyicon.net.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="btnToDetectSampleImgs">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>检测图片</string>
            </property>
            <property name="icon">
             <iconset resource="findcirclesapplication.qrc">
              <normaloff>:/检测图片/tupian/检测图片/face_detect_43.467811158798px_1195571_easyicon.net.png</normaloff>:/检测图片/tupian/检测图片/face_detect_43.467811158798px_1195571_easyicon.net.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="editResult">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelShowResult">
            <property name="text">
             <string>显示结果</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnNextImg">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>下一张</string>
          </property>
          <property name="icon">
           <iconset resource="findcirclesapplication.qrc">
            <normaloff>:/下一张/tupian/下一张/next_48px_1205821_easyicon.net.png</normaloff>:/下一张/tupian/下一张/next_48px_1205821_easyicon.net.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExit">
          <property name="text">
           <string>退出</string>
          </property>
          <property name="icon">
           <iconset resource="findcirclesapplication.qrc">
            <normaloff>:/退出/tupian/退出/system_exit_48px_540086_easyicon.net.png</normaloff>:/退出/tupian/退出/system_exit_48px_540086_easyicon.net.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1333</width>
           <height>577</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="ImgLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>显示图片</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1367</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="findcirclesapplication.qrc"/>
 </resources>
 <connections/>
</ui>
